tests/analyser/analyser.cpp

```diff
@@ -74,17 +68,14 @@ TEST(Analyser, twoVariablesOfIntegration)
     EXPECT_EQ(size_t(0), parser->issueCount());
 
     const std::vector<std::string> expectedIssues = {
-        "Variable 'time' in component 'main' of model 'two_variables_of_integration' and variable 'other_time' in component 'sub_sub_sub' of model 'two_variables_of_integration' cannot both be the variable of integration.",
-    };
-    const std::vector<libcellml::Issue::Cause> expectedCauses = {
-        libcellml::Issue::Cause::UNDEFINED,
```

Comment:
Cause should be VARIABLE?

Reply:
You commented on something that is outdated...

Reply:
FWIW, the test doesn't check the cause anymore, although it is / should be undefined here, I think. I mean, it's not related to a variable as such...?

Reply:
> You commented on something that is outdated... 

I went "see changes since your last review" and commented on those?  Not sure why that would show something outdated?

I don't understand the point of having both an analyser class and a set of issue causes if we don't actually use them together?  What's the cause if not the variable(s)?

Reply:
Not sure what you did / what happened with GitHub, but the line you commented on doesn't exist anymore... :)

Reply:
> I don't understand the point of having both an analyser class and a set of issue causes if we don't actually use them together? What's the cause if not the variable(s)?

I thought that the cause only made sense in the context of validation, but happy to put back a more meaningful cause. I have no problems with that. Ok, am going to do it then.

Reply:
Ok, this has been done in commit c78a14c9